R/utils.R

```diff
@@ -351,6 +351,46 @@
     altExps(x) <- values
     return(x)
 }
+
+# This function can be used to add tree to existing tree
+.add_tree <- function(x, tree, MARGIN, name = "phylo", ...) {
```

Comment:
So:

1. Add all the functionality to .add_tree() function.
2. Check MARGIN and name
3. Get correct functions based on MARGIN
4. Check if tree name. If it does exist, give warning
5. Add tree to object

changeTree() seems not to work when the tree does not already exist with that name.